### DISCUSSION TITLE
map.flyTo with elevation

### DISCUSSION THREAD

**Discussion by MrMorice** [maplibre/maplibre-gl-js#2989] [Q&A]
Hi everybody, great library 👌. I'm writing an application with maplibre with terrain and a deck.gl overlay with flying objects (paragliders). Looks pretty good but when I use map.flyTo, the objects will usually be on top or out of the viewport because they have an elevation above terrain.

Currently flyTo seems to use the terrain elevation for calculating the viewport center. Is there a way to extend this behaviour and use a known elevation instead?

Thanks and best, Mo

**Comment by HarelM** [comment]
You might have more luck posting this in the slack channel.
Cc @birkskyum
